content/applications/hr/time_off/management.rst

```diff
@@ -0,0 +1,150 @@
+==========
+Management
+==========
+
+.. _time_off/approvals:
+
+Time off and allocation requests typically undergo an approval process before being granted.
+Requests either need one or two approvals, if any, depending on how the specific type of time off is
```

Comment:
We either need an approval or we don't, please pick one ;)
As it's written, this is contradictory to itself.

Reply:
Agreed! When a manager requests time off, they can technically approve it themselves if they are the approver. That's what happens in Runbot (and that's an insane way to manage time off) which is why this came about originally.